src/Console/Command/Option/HostNameOptionTrait.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace duckpony\Console\Command\Option;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @uses \Zend\Config\Config
+ * @author Dogan Ucar <dogan@dogan-ucar.de>
+ */
+trait HostNameOptionTrait
+{
+
+    protected function configureHostNameOption(): void
+    {
+        $this->addOption('host_name', 'hn', InputOption::VALUE_OPTIONAL, 'The hostname of the master host machine');
```

Comment:
I think given this option the `InputOption` value needs to be required.
As it doesn't make sense to leave it empty.

Reply:
Also. I don't think the short handle with two characters works. Did you check it?

Reply:
Thank you for your feedback. I changed the option's value to required. I missunderstood the `optional`, sorry!

Indead, the two charactered short handle does not work. Changed it to `h`, if this is ok for you.